WTWSMS/history/characters/WTWSMS_enlil.txt

```diff
@@ -7612,6 +7635,7 @@
 		birth = yes
 	}
 	610.1.1 = {
+>>>>>>> master
 		death = yes
 	}
```

Comment:
There are two conflicts here, caused by the "cleaning" in the master. Either you can overwrite the master or merge properly (the "cleaning" has no impact on the code so can technically be ignored, but that means @IhateTrains would have to redoit from scratch).

Reply:
I think I made it

Reply:
The cleaning takes a few seconds, so I don't really care.

Reply:
Don´t worry, I think it is solved